pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -285,18 +1104,33 @@ func TestListUsers(t *testing.T) {
 		ctx := typesystem.ContextWithTypesystem(context.Background(), typesys)
 
 		t.Run(test.name, func(t *testing.T) {
```

Comment:
i would move all the code related to each test (lines 1091-1102) inside `t.Run`, otherwise if some of those lines throw an error it will be harder to say to which test they belong